lib/server/server.js

```diff
@@ -100,11 +109,14 @@ export const startServer = async (serverConfig) => {
       const httpServer = http.createServer(app);
 
       // Attach error handlers and listen to the server
-      attachErrorHandlers(httpServer);
+      attachServerErrorHandlers(httpServer);
 
       // Listen
       httpServer.listen(serverConfig.port, serverConfig.host);
 
+      // Save the reference to HTTP server
+      activeServers.push(httpServer);
```

Comment:
I'm not sure if that'll ever be the case (maybe I'm missing something here), but I see that we do `activeServers.push(httpServer)`, but we never clean the `activeServers`. Even `server.on('close', () => ...)` we only log that the server has been closed. Maybe if there are multiple active servers and one of them gets closed we should remove it from the array?

Reply:
@jszuminski Actually, we do close servers. Take a look at the `shutdownCleanUp` function, where servers are get with the `getServers`.